package.json

```diff
@@ -101,6 +101,12 @@
       "default": false,
       "order": 5
     },
+    "enableSynctex": {
+      "title": "Enable SyncTeX",
```

Comment:
Sorry, you are entirely correct! Was confusing myself thinking you were changing the description, but there is no description.